silk/migrations/0010_fill_query_details.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.2.24 on 2021-06-24 09:54
+
```

Comment:
is this migration handwritten? or auto generated

Reply:
The code inside the migration was handwritten, the file was autogenerated using `./manage.py makemigrations silk --empty --name fill_query_details`